server/shared/src/main/scala/org/http4s/server/ContextRouter.scala

```diff
@@ -81,17 +81,38 @@ object ContextRouter {
   /** Defines an [[ContextRoutes]] based on list of mappings.
     * @see define
     */
-  def apply[F[_]: Sync, A](mappings: (String, ContextRoutes[A, F])*): ContextRoutes[A, F] =
+  def apply[F[_]: Monad, A](mappings: (String, ContextRoutes[A, F])*): ContextRoutes[A, F] =
     define(mappings: _*)(ContextRoutes.empty[A, F])
 
+  @deprecated("Kept for binary compatiblity.  Use the apply with the Monad constraint.", "0.23.13")
```

Comment:
Right? So hard to keep track 😆 

```suggestion
  @deprecated("Kept for binary compatiblity.  Use the apply with the Monad constraint.", "0.23.12")
```